### PR TITLE
Feature/tnlt 9632 pass utm values to native module analytics

### PR DESCRIPTION
**Pull request by petermason**
#### Description

If the Topic and Author page is accessed via deeplink return the deeplink parameters back as a tracking event.

#### Checklist
 - [x] Unit tests
 - [ ] Updated E2E tests
 - [ ] Updated storybook
 - [ ] Tested on iOS simulator
 - [ ] Tested on Android emulator
 - [ ] Tested on Tablet

### CODE REVIEW

File: packages/pages/src/author-profile/index.js [hunk 1]
```diff
@@ -11,7 +11,19 @@ const {
...
+    if (deeplink_value) {
+      track({
+        attrs: {
+          eventTime: new Date(),
+          pageName: "AuthorProfile",
+        },
+        ...deeplink_value,
+      });
+    }
+  }, deeplink_value);
```
Comment (josholdham) [inline packages/pages/src/author-profile/index.js:25]: Genuine question: does this work correctly when the dependency (deeplink_value) isn't part of an array?

I assume so, but just not seen it before

**Review by josholdham — APPROVED** [review]
Good old deeplinks!

I have a couple of comments/questions, but no blockers, so approved ✌️

File: packages/pages/src/article/index.js [hunk 2]
```diff
@@ -24,6 +24,8 @@ const ArticlePage = (props) => {
     onAction: openInBrowser,
   };
 
+  console.log("ARTICLE PROPS: ", props);
```
[the remark this answers is not in the document]
Comment (petermason) [reply packages/pages/src/article/index.js:27]: oops :)

File: packages/pages/src/author-profile/index.js (hunk 1 above)
Comment (petermason) [reply packages/pages/src/author-profile/index.js:25]: actually, this *should* be in an Array! good catch

File: packages/pages/src/article/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (petermason) [reply packages/pages/src/article/index.js]: removed

File: packages/pages/src/author-profile/index.js (hunk 1 above)
Comment (petermason) [reply packages/pages/src/author-profile/index.js]: updated to correct syntax

**Comment by josholdham** [comment]
All looks good - nice one

**Review by josholdham — APPROVED** [review]

**Pull request merged**